app/models/user.rb

```diff
@@ -1,4 +1,9 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Hi as mentioned by the previous reviewer, Please ass `:confirmable` to your device settings

Reply:
- [ ] Hello. Please as mentioned by the previous reviewers, you have to add `:confirmable` to your devise settings.